### ISSUE TITLE
How to set create static pages in the top-right menu.

### ISSUE THREAD

**Issue by willismonroe** [PippoRJ/hugo-refresh#3]
I saw from issue #1 that the top-right menu is created programmatically based on the contents of "/content/".  I'm trying to create a page that appears in that list but is just a static .md file.

If I leave a "_index.md" file in the directory I get the name of the directory in the menu but no content.  If I change the "_index.md" to just "index.md" I can show content but the link disappears from the menu.  If I have both, the "index.md" appears as a box but the link just reloads the page.

**Comment by willismonroe** [comment]
Ah I figured it out, I didn't need to have the folder hierarchy.  If I just added a named md file to the "/content/" directory it showed up as a link in the top-right menu.

**Comment by willismonroe** [comment]
Re-opening, because I was mistaken, after reloading the links are gone from the top-right menu.  
![image](https://user-images.githubusercontent.com/715210/65842202-a7f55100-e2de-11e9-9306-027c81783a47.png)
The layout above results in the page below (I'd like publications and teaching to appear in the menu):
![image](https://user-images.githubusercontent.com/715210/65842206-af1c5f00-e2de-11e9-96cc-900bb2eb4da3.png)

This issues has been a roller-coaster...

**Comment by PippoRJ** [comment]
Hi @willismonroe ,

you are right, at this moment, in the top right menu you cannot have regular pages. 
In the "content" folder you can have only other folders that will contain regular pages.

I really need to add this as you suggest. I will do it in the next few days!

**Comment by willismonroe** [comment]
That would be amazing, let me know if there's anything I can do to help.

**Comment by PippoRJ** [comment]
I added that functionality.

In any regular page you can now specify the option `showInMenu`.
If the option `showInMenu` is set to `true` the page will be shown in the top-right menu.

Please let me know if everything is working.

**Comment by willismonroe** [comment]
Hey I'm excited to give this a shot, but I ran a git pull on the submodule theme and now I'm getting this error when I try to serve the site:
`
 parse failed: template: partials/css.html:27: illegal number syntax: "-"
`
(This is hugo version: `Hugo Static Site Generator v0.58.0/extended linux/amd64 BuildDate: 2019-10-03T11:09:47Z`)

I had a look at the file and I'm not sure what could be wrong, it's in the middle of other lines that also have that "-" character (which seems to be valid for templates of course).

**Comment by PippoRJ** [comment]
mmm, that's odd.

I will be offline for 2 weeks unfortunately but I will have a look as soon
as I am back online.

Sorry for keep you waiting..!

**Comment by willismonroe** [comment]
No worries, enjoy being off-line.

**Comment by willismonroe** [comment]
For when you return, I've found that this commit (https://github.com/PippoRJ/hugo-refresh/commit/6a6ff4cdbb2bf3fbd4a950db1937c07435b386bb) is the one breaking it for me.  Again, I don't really see what could be wrong there... but luckily the single pages change you made is before that, so I can keep working with that.

**Comment by PippoRJ** [comment]
Hi @willismonroe ,
I was not able to reproduce the error, with my website and with the example site it is working... not sure what is happening there.
Anyway, I removed the extra spaces.
Let me know if you can update the theme and run it without problems..!

**Comment by vBuresh** [comment]
Hi, thanks for the great theme and the ability to display the avbranny ordinary pages in the menu. 
Is it possible to allow translation of the generated menu items into another language?

**Comment by PippoRJ** [comment]
Hi @vBuresh 
I am not an expert of the Multilingual Mode of Hugo but it is certainly possible:
https://gohugo.io/content-management/multilingual/
Did you know about this feature? Does it work for you?

I did not have the multilingual mode in mind when developing the theme but I will be happy to know if it works... and if it doesn't I can have a look into how to make it work..!

**Comment by vBuresh** [comment]
Hi Roberto!

Thanks for your prompt reply and helpfulness.
So far, creating multilingual sites for me is tomorrow's task, but it is very interesting. I look at her, including [discussions ](https://discourse.gohugo.io/t/translation-of-the-plural-and-zero-in-russian-returns-the-name-of-the-variable/14995).

But now I need translation of a few interface elements only:
- Main menu (usually 2-5 positions)
- other items (5 positions)
    1. /layouts/partials/single/summary.html  
    **Read More** to **Читать...**
    2. /layouts/partials/single/summary.html  
    **Read More** to **Читать...**
    3. /layouts/partials/single/navbar-clone.html  
    /layouts/partials/navbar.html
    **Menu** to **Меню**
    4. /layouts/partials/single/summary.html  
    **All Tags** to **Все теги**
    5. /layouts/partials/single/content.html  
    **Last modified:** to **Обновлено:**

To translate user interface terms I crawled into the source code. But this is the wrong method. Better to make the settings. Unfortunately, my skills do not yet allow me to do this myself.

However, I tried to make the necessary changes in the code (except for menu items - by the names of the / content subdirectories). It works. Only when scrolling there are arbitrary switching between **Menu** and **Меню** 

And yet, it’s important to have a date format setting for: 
`Last modified: {{- .Lastmod.Format "2 January 2006" -}}`

Thank you again for the wonderful, very concise, elegant theme.
